3/ollie-charles-captions.srt

```diff
@@ -2299,12 +2301,11 @@ So let's filter the dictionary
 
 531
 00:34:20,680 --> 00:34:23,000
-for stuff that's not to equal to N.
+for stuff that's not equal to N.
 
 532
 00:34:28,280 --> 00:34:31,880
-It's still having (inaudible)
-
+It's still compiling...
```

Comment:
In Vim, this lovely regex matches captions that are not seperated from their predecessor with a newline:

    \S\n\zs\d\+\n